shared/actions/teams/index.js

```diff
@@ -75,6 +76,19 @@ const _addPeopleToTeam = function*(action: Constants.AddPeopleToTeam) {
   yield put((dispatch: Dispatch) => dispatch(Creators.getDetails(teamname))) // getDetails will unset loading
 }
 
+const _inviteByEmail = function*(action: Constants.InviteToTeamByEmail) {
+  const {payload: {invitees, role, teamname}} = action
+  yield put(replaceEntity(['teams', 'teamNameToLoading'], I.Map([[teamname, true]])))
+  yield call(RpcTypes.teamsTeamAddEmailsBulkRpcPromise, {
```

Comment:
:sheep: This should also maybe be in a `try` like L81 & others that set `teamNameToLoading` to make sure it gets unset. Though with `allowOwner: false` this shouldn't fail